doc/api/cli.md

```diff
@@ -971,11 +971,15 @@ for the very first unhandled rejection in case no [`unhandledRejection`][] hook
 is used.
 
 Using this flag allows to change what should happen when an unhandled rejection
-occurs. One of three modes can be chosen:
+occurs. One of the following modes can be chosen:
 
+* `throw`: Emit [`unhandledRejection`][]. If this hook is not set, raise the
+  unhandled rejection as an uncaught exception.
 * `strict`: Raise the unhandled rejection as an uncaught exception.
 * `warn`: Always trigger a warning, no matter if the [`unhandledRejection`][]
   hook is set or not but do not print the deprecation warning.
+* `warn-with-error-code`: Emit [`unhandledRejection`][]. If this hook is not
```

Comment:
Sorry for bikeshedding the name further but what about: `warn-and-exit`?

Reply:
I'm OK with a bit more bikeshedding on this, but be advised that I'd already filed a PR to update the name in the unhandled rejections survey. https://github.com/nodejs/TSC/pull/872 It 

IMO, `warn-and-exit` sounds like it will actually terminate the process, but this flag is designed to set `process.exitCode = 1` _and keep running_. The exit code may never even take effect if it's a long-running process that never terminates until killed.

I think `warn-with-error-code` is the shortest and clearest way to say, "trigger a warning, and set the process exit code to 1"

Reply:
@dfabulich I don't think it's a problem if we end up with different names on the survey, because we explain what each mode is there (and we're only using mode names there as an alias for what they mean).

Reply:
For my part, I can’t think of a better name for this than `warn-with-error-code`. Let’s merge this.

Reply:
Is it time?

Reply:
It's ready to land. I'm very absent on GitHub due to everything that is happening, but another collaborator might land it.

Reply:
Would it help if this PR had the "author ready" label?